homeassistant/components/uptimerobot/config_flow.py

```diff
@@ -43,14 +45,14 @@ async def async_step_user(self, user_input: ConfigType | None = None) -> FlowRes
             )
 
         try:
-            await validate_input(self.hass, user_input)
+            account = await validate_input(self.hass, user_input)
         except CannotConnect:
             errors["base"] = "cannot_connect"
         except Exception:  # pylint: disable=broad-except
             LOGGER.exception("Unexpected exception")
             errors["base"] = "unknown"
         else:
-            return self.async_create_entry(title="", data=user_input)
+            return self.async_create_entry(title=account.email, data=user_input)
```

Comment:
For the future, maybe we can use the email as config entry unique_id?

Reply:
Config entries are unique?

Reply:
They themselves are, yet we can make them more unique for the service-connected, e.g., by setting a unique ID from upstream.

This helps to prevent adding multiple devices/services twice for the same device/account. It also helps with things like re-authentication.

Reply:
If there's a valid unique id we can use the config entries api for unique id which makes it easier to avoid duplicate config entries, update existing entries with new data etc.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids

Reply:
Since the user can change the email, I would not consider that to be stable enough.
Authentication is with an api_key only, (re-authentication is planed for the next PR against this integration)
I'll see if there is an account ID that can be used if deciered

Reply:
Added `user_id` support in 21.8.1, will do a follow-up to implement that, this one is big enough already :+1: